packages/pds/src/pipethrough.ts

```diff
@@ -104,11 +64,11 @@ const REQ_HEADERS_TO_FORWARD = [
 export const createUrlAndHeaders = async (
   ctx: AppContext,
   req: express.Request,
-  requester?: string,
+  requester: string | null,
   audOverride?: string,
 ): Promise<{ url: URL; headers: { authorization?: string } }> => {
   const proxyTo = await parseProxyHeader(ctx, req)
-  const defaultProxy = defaultService(ctx, req.path)
+  const defaultProxy = defaultService(ctx, req.baseUrl)
```

Comment:
The mount path doesn't get included in `req.path` (https://expressjs.com/en/4x/api.html#req.path) so had to change to `baseUrl`

Reply:
I think this works due to an implementation detail of xrpc-server, that it happens to write its catchall route this way:
```ts
    this.router.use(
      '/xrpc/:methodId', // <- baseUrl comes from here
      opts?.catchall ?? this.catchall.bind(this),
    )
```

If we're able to base this on `req.originalUrl` (need to account for it containing query params) I think we may be able to avoid a potential footgun.  Alternately can probably carefully concat `req.baseUrl` and `req.path` together.

Reply:
yup yup totally 👍 

ended up rewriting to use `req.originalUrl`

also fwiw, this logic is only used to find the default service so it's relatively low stakes 👌